operator-metadata/manifests/bundle.clusterserviceversion.yaml

```diff
@@ -1190,7 +1206,7 @@ spec:
           replicas: 1
           selector:
             matchLabels:
-              name: amq-streams-cluster-operator
+              name: strimzi-cluster-operator
```

Comment:
We have `amq-streams-cluster-operator` in all previous releases, or?

Reply:
Correcy. Now fixed